referenceframe/model.go

```diff
@@ -296,3 +308,32 @@ func ModelFromPath(modelPath, name string) (Model, error) {
 	}
 	return model, err
 }
+
+// New2DMobileModelFrame builds the kinematic model associated with the kinematicWheeledBase
```

Comment:
Can we add more details on exactly what the structure of this model is?

Reply:
Made a better comment